pkg/sql/logictest/testdata/logic_test/show_table

```diff
@@ -0,0 +1,17 @@
+# LogicTest: local
+
+statement ok
+SET CLUSTER SETTING sql.show_tables.show_estimated_row_count = false
```

Comment:
i guess it has to be called `sql.show_tables.estimated_row_count.enabled` ? weird naming ergonomics..